core/src/main/groovy/com/novoda/gradle/release/AndroidLibrary.groovy

```diff
@@ -5,13 +5,14 @@ import org.gradle.api.Project
 import org.gradle.api.artifacts.Dependency
 import org.gradle.api.artifacts.ModuleDependency
 import org.gradle.api.artifacts.PublishArtifact
+import org.gradle.api.attributes.Usage
 import org.gradle.api.internal.DefaultDomainObjectSet
 import org.gradle.api.internal.component.SoftwareComponentInternal
-import org.gradle.api.internal.component.Usage
+import org.gradle.api.internal.component.UsageContext
 
 class AndroidLibrary implements SoftwareComponentInternal {
 
-    private final Usage runtimeUsage
+    private final UsageContext runtimeUsage
```

Comment:
Are these changes going to work if the users of the plugin have gradle version older than 3.5? I guess with this, the library will require the users to use version 3.4?